src/styles/treemap.scss

```diff
@@ -4,13 +4,13 @@
 }
 
 .rv-treemap__leaf {
-  position: absolute;
+  border: thin solid #fff;
```

Comment:
Does this style change have any implication downstream? I'm concerned that a library downstream now sees a border where there wasn't one before.